### ISSUE TITLE
support private github overrides without prompting for user/pass

### ISSUE THREAD

**Issue by binary1230** [magfest/ubersystem-deploy#11]
When using custom private repos in fabric_settings.ini, ./setup_vagrant_control_server will prompt you for a username and password for github.   

we should make it so that an SSH key (or something) can be put in place before starting the deploy to ensure the entire deploy runs in a completely automated fashion.

low priority thing.
